showcase/app/styles/showcase-pages/advanced-table.scss

```
@@ -83,4 +83,10 @@ body.page-components-advanced-table {
     white-space: nowrap;
     text-overflow: ellipsis;
   }
+
+  .shw-component-advanced-table-reorder-handle-container
+    .hds-advanced-table__th-reorder-handle {
+    visibility: visible;
+    opacity: 1;
```

```suggestion
      opacity: 1;
```
[Nit] Fix style indentation